gfx_scene.h

```diff
@@ -1773,14 +1765,23 @@ class GfxSceneInternal
                 }
             }
             std::set<GfxConstRef<GfxAnimation>> children_animations;
+            std::set<GfxConstRef<GfxAnimation>> propogate_parent_animations = parent_animations;
+            if (node_animations.find(gltf_node) != node_animations.end())
+            {
+                propogate_parent_animations.insert(
+                    node_animations[gltf_node].begin(), node_animations[gltf_node].end());
+                children_animations.insert(
+                    node_animations[gltf_node].begin(), node_animations[gltf_node].end());
+            }
             for(size_t i = 0; i < gltf_node->children_count; ++i)
             {
-                auto child_animations = VisitNode(gltf_node->children[i], transform, node_animations[gltf_node]);
+                auto child_animations =
+                    VisitNode(gltf_node->children[i], transform, propogate_parent_animations);
```

Comment:
Need to propogate the current node transform as well as the parent tansform to the children otherwise parent animations are lost and not all child nodes are created